index.js

```diff
@@ -18,20 +18,37 @@ bedrock.events.on('bedrock-cli.init', () => bedrock.program
   .option('-h, --hostname <value>', 'The ledger host.')
   .option('-k, --ignoreSslErrors', 'Use strict SSL.')
   .option('-s, --send', 'Send a demonstration operation.')
+  .option('-r, --recordPath <value>', 'A path to a json record to be sent.')
```

Comment:
would `recordJson` or `recordFile` be better?

Reply:
file might be good Json I think implies we passing a string that can be json.

Reply:
addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd